### ISSUE TITLE
Alt text

### ISSUE THREAD

**Issue by farishkash** [rmcknight/Colmar#4]
https://github.com/rmcknight/Colmar/blob/e161266dbbcd706d4b1552c33324d486bf8fb3ba/index1.html#L51

For alt text don't use unable to load image.  Use a detailed description instead.  The reason being is that the visually impaired use screen readers and the alt text is their source of information for that image.
